book/mars_maps_extended.ipynb

```diff
@@ -0,0 +1,600 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Making some Mars maps with pygmt\n",
+    "\n",
+    "This tutorial page covers the basics of creating some maps and 3D plot of Mars (yes! Mars). The idea here is to demonstrate that you can use a simple sequence of commands with PyGMT, a Python wrapper for the Generic Mapping Tools (GMT), and with some public data about the topography of Mars, create your own maps, as well as compare this topography with what we know of our own planet Earth.\n",
+    "\n",
+    "## first, some options\n",
+    "\n",
+    "You can run this notebook using your local pygmt installation, or via Binder, or even Google Colaboratory. See comments for each option below. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**A)** A short note if you are using COLAB\n",
+    "\n",
+    "The version of python in COLAB is different from what the newer GMT needs to install along with pygmt. So, one way around this problem is to reinstall GMT from scratch, along with other important packages. This is done with this block of commands below.\n",
+    "\n",
+    "**comment out the first line of the block (%%script echo skipping) if you want to use colab**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "nnMDokWZuQyN",
+    "outputId": "0be69909-5cb9-4d8c-acfb-1b0b6b303b8f"
+   },
+   "outputs": [],
+   "source": [
+    "%%script echo skipping\n",
+    "\n",
+    "# because I like to enjoy my coffee in silence, it takes time.  \n",
+    "# (3 runs averaged 6 minutes to install everything ! keep drinking your coffee)\n",
+    "# comment the %%capture line if you want to see the colab VM working\n",
+    "%%capture\n",
+    "!sudo apt update \n",
+    "!sudo apt upgrade -y\n",
+    "!sudo apt install -y build-essential cmake libcurl4-gnutls-dev libnetcdf-dev gdal-bin libgdal-dev libfftw3-dev libpcre3-dev liblapack-dev libblas-dev libglib2.0-dev ghostscript ghostscript-x graphicsmagick ffmpeg xdg-utils\n",
+    "# clone gmt from source\n",
+    "!git clone --depth 50 https://github.com/GenericMappingTools/gmt\n",
+    "# cmake everything\n",
+    "!cmake /content/gmt\n",
+    "# build and install\n",
+    "!cmake --build . --target install\n",
+    "\n",
+    "# and last but not least\n",
+    "!pip install pygmt\n",
+    "\n",
+    "# and if you don't believe in it\n",
+    "!gmt --version\n",
+    "!python --version"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "oIdVNbxrr1lw",
+    "outputId": "05379551-c66c-4d0b-917c-74cc25fb8942"
+   },
+   "outputs": [],
+   "source": [
+    "%%script echo skipping\n",
```

Comment:
```suggestion
```
I think this cell magic will need to be removed for the jupyter book to deploy properly